features/command_line_interface/options.feature

```diff
@@ -41,7 +41,8 @@ Feature: Reek can be controlled using command-line options
 
       Configuration:
           -c, --config FILE                Read configuration options from FILE
-              --smell SMELL                Detect smell SMELL (default: all enabled smells)
+              --smell SMELL                Detect smell or smell category SMELL
```

Comment:
Hmm, how about adding <>, so <SMELL> to make it more obvious?
Right now it reads like a typo.